local-tests/setup/shiva-client.ts

```diff
@@ -102,100 +110,82 @@ export class TestnetClient {
       const pollRes = await fetch(
         this._processEnvs.TESTNET_MANAGER_URL + '/test/poll/testnet/' + this._id
       );
-      const res = await pollRes.json();
+      const res: TestNetResponse<TestNetState> = await pollRes.json();
```

Comment:
Should this be above L113's attempt to parse the result as json? If we get an error 500 from a request, it probably won't be in JSON, no?  Maybe this should use `_processTestnetResponse()` also?